lib/ecs_deploy/service.rb

```diff
@@ -110,6 +110,7 @@ def deploy
 
         service_options.merge!({service: @service_name})
         service_options.merge!({desired_count: @desired_count}) if @desired_count
+        service_options.merge!({propagate_tags: @propagate_tags}) if @propagate_tags
```

Comment:
[nits] An AWS client ignores nil values, so you can assign `propagate_tags` when defining `service_options`.